packages/cli/test/utils/simulation/validator_clients/lighthouse.ts

```diff
@@ -39,7 +39,6 @@ export const generateLighthouseValidatorNode: ValidatorNodeGenerator<ValidatorCl
     "beacon-nodes": beaconUrls[0],
     "debug-level": "debug",
     "init-slashing-protection": null,
-    "allow-unsynced": null,
```

Comment:
This param is not supported with new release of lighthouse.

Reply:
Looking at the [latest run](https://github.com/ChainSafe/lodestar/actions/runs/7748011734/job/21129738871#step:8:11) on unstable branch, I don't see any error that would indicate that Lighthouse failed to start. Where would I find this, just in the log files? I feel like a service failing to start should be printed out in the console.

> This param is not supported with new release of lighthouse

There was some discussion on this in https://github.com/ChainSafe/lodestar/issues/5942, I'd agree with Lion here that we use fixed versions everywhere, our CI needs to be deterministic.

And then we need to set up a CI run that tests against latest version
> To ensure that Lodestar works with new releases there should be a new workflow that runs on a cron job (say, daily) against un-pinned version

Reply:
> Looking at the [latest run](https://github.com/ChainSafe/lodestar/actions/runs/7748011734/job/21129738871#step:8:11) on unstable branch, I don't see any error that would indicate that Lighthouse failed to start. Where would I find this, just in the log files? I feel like a service failing to start should be printed out in the console.

Lighthouse Validator could not start in the mixed clients tests. In the logs file saw the error.